README.md

```diff
@@ -96,6 +98,49 @@ which will enable you to call the container like:
 sdcli go test
 ```
 
+## For Shells Other than `bash`
+
+In fish shell, you create a `~/.config/fish/functions/sdcli.fish` file with 755 permissions having contents:
+
+```bash
+function sdcli
+  set cwd (pwd -L)
```

Comment:
my shell complains that there is no -L option:

   Description
       pwd outputs (prints) the current working directory.

       Note that fish always resolves symbolic links in the current directory path.